Camera2SlowMotion/app/src/main/java/com/example/android/camera2/slowmo/fragments/CameraFragment.kt

```diff
@@ -71,6 +70,11 @@ import kotlin.coroutines.suspendCoroutine
 
 class CameraFragment : Fragment() {
 
+    /** Android ViewBinding */
+    private var _fragmentCameraBinding: FragmentCameraBinding? = null
+
```

Comment:
This structure doesn't seem necessary. You could replace with `private lateinit var fragmentCameraBinding: FragmentCameraBinding` since the Fragment's lifecycle protects against needing the `_fragmentCameraBinding`